### ISSUE TITLE
CDS file in panEDTA

### ISSUE THREAD

**Issue by Isoris** [oushujun/EDTA#431]
Hello, 

Is it possible to use multiple CDS files from genbank and to combine them into a single file for panEDTA ? 

for instance cat cds_species1.fa cds_species2.fa cds_species3.fa > cds.fa 

When running panEDTA with multiple species, it should be fine right because it's basically a blast to whitelist CDS? 

Thank you for your answer.

**Comment by oushujun** [comment]
Yes, this is possible, but keep in mind that when you aggregate things, both the good and the bad will be aggregated. If you have TE contaminations in your CDS files, they will add up and reduce your library sensitivity.

Shujun
